recipes/rhizopus/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "rhizopus" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jwergieluk/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a407e1a9d82816c6a1b116a960858a524458cb0734ac4a7a1b21ab5e6365ce8a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  commands:
+    - python --version
+
+about:
+  home: https://github.com/jwergieluk/rhizopus
+  license: GPL-3.0-or-later
+  license_family: GPL3
+  license_file: LICENSE
+  summary: 'Trading simulator'
+  description: |
+    rhizopus is a Python trading simulation framework and a backtesting tool.
+    It can be used to construct broker simulators for backtesting with
+    historical data, as well as for live trading. Its main goal is to provide a
+    simple unified interface for both backtesting and live trading.
+  doc_url: https://github.com/jwergieluk/rhizopus
+  dev_url: https://github.com/jwergieluk/rhizopus
+
+extra:
+  recipe-maintainers:
+    - jwergieluk
+    - shackmann
```

Comment:
@shackmann Are you happy to be listed as a maintainer here?

Reply:
Hi. Yes, please add me too. Thank you!